posts/views.py

```diff
@@ -14,6 +14,14 @@ def post_list(request):
     return Response(posts, status=status.HTTP_200_OK)
 
 
+@api_view(['GET'])
+@permission_classes([p.IsAuthenticated, ])
+def users_posts(request):
+    user = request.user
+    posts = Post.objects.filter(is_deleted=False, author_id=user).values()
```

Comment:
You will need to add pagination because now you get all user posts

Reply:
Done.

Reply:
You don't need another endpoint, you need to update the users_posts endpoint. And "user-posts" naming  look better